updated discover with alt text (#77660)
elastic/kibana: docs/canvas/canvas-share-workpad.asciidoc

Click *Share > Download as JSON*.

[role="screenshot"]

> image::images/canvas-export-workpad.png[Export single workpad through JSON, from "share" dropdown]

Reviewer: ```suggestion
image::images/canvas-export-workpad.png[Export single workpad through JSON, from Share dropdown]
```

Click *Share > PDF reports > Generate PDF*.

[role="screenshot"]

> image::images/canvas-generate-pdf.gif[Image depicting generate pdf option]

Reviewer: ```suggestion
image::images/canvas-generate-pdf.gif[Image showing how to generate a PDF]
```
